internal/app/export/service.go

```diff
@@ -109,20 +109,27 @@ func (e *exporter) Start(queue string) error {
 	return nil
 }
 
-func (e *exporter) Publish(subject, topic string, payload []byte) {
+func (e *exporter) Publish(subject, topic string, payload []byte) errors.Error {
 	e.logger.Debug(fmt.Sprintf("Publishing to topic %s", topic))
-	if err := e.publish(topic, payload); err == nil {
-		return
+	err := e.publish(topic, payload)
+	if err != nil {
+		e.logger.Error(err.Error())
+	}
+	if err == nil {
```

Comment:
Why is this check needed?

Just move the blow `e.cache` check and other stuff in the upper `if err != nil` block and do not check nothing here, just return `nil`